orienteer-core/src/main/java/org/orienteer/core/component/table/filter/ODocumentProvider.java

```diff
@@ -0,0 +1,60 @@
+package org.orienteer.core.component.table.filter;
+
+import com.orientechnologies.orient.core.record.impl.ODocument;
+import org.apache.wicket.extensions.markup.html.repeater.data.table.filter.IFilterStateLocator;
+import org.apache.wicket.extensions.markup.html.repeater.util.SortableDataProvider;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.Model;
+import ru.ydn.wicket.wicketorientdb.model.OQueryDataProvider;
+
+import java.util.Iterator;
+
+/**
+ * @author Vitaliy Gonchar
+ */
+public class ODocumentProvider extends SortableDataProvider<ODocument, String>
```

Comment:
Imho, this stuff should be in wicket-orientdb on OQueryDataProvider. Can you please split functionality and PR to wicket-orientdb required things?